app/controllers/api/base_controller.rb

```diff
@@ -35,6 +35,15 @@ def render_400(type)
       render_error(400, type)
     end
 
+    def render_failed_validations(errors)
+      render json: {
+        error: {
+          type: :failed_validations,
+          message: I18n.t("api.errors.failed_validations", errors: errors.full_messages.join(", "))
+        }
+      }, status: :bad_request
```

Comment:
I can push this tomorrow, but if you want to do it, I'd do it as:

```suggestion
    def render_failed_validations(errors)
      render json: {
        error: {
          type: :failed_validations,
          message: I18n.t("api.errors.failed_validations"),
          errors: errors.full_messages
        }
      }, status: :bad_request
```

Also, if you've not seen, validation errors have changed lots in Rails 6.1. They're now rich objects. Probably worth looking at! :)

Reply:
Thanks :) I've done this now.